public/components/common/header.tsx

```diff
@@ -20,28 +20,17 @@ export const Header = ({ children }: HeaderProps) => {
       style={{ borderBottom: '1px solid #D3DAE6' }}
     >
       <EuiPageHeader pageTitle="Compare search results">
-        <EuiCallOut title="Experimental Feature" iconType="iInCircle">
-          <EuiText>
-            <p>
-              Compare results using the same search text with different queries. For more
-              information, see the{' '}
-              <EuiLink
-                href="https://opensearch.org/docs/latest/search-plugins/search-relevance"
-                target="_blank"
-              >
-                Compare Search Results Documentation
-              </EuiLink>
-              . To leave feedback, visit{' '}
-              <EuiLink
-                href="https://forum.opensearch.org/t/feedback-experimental-feature-compare-search-results/11331"
-                target="_blank"
-              >
-                forums.opensearch.com
-              </EuiLink>
-              .
-            </p>
-          </EuiText>
-        </EuiCallOut>
+        <EuiText>
+          <p>
+            Compare results using the same search text with different queries.{' '}
+            <EuiLink
+              href="https://opensearch.org/docs/latest/search-plugins/search-relevance"
+              target="_blank"
+            >
+              Learn more
+            </EuiLink>
+          </p>
+        </EuiText>
```

Comment:
Can we make the space between the title and the description seem like this? https://github.com/opensearch-project/dashboards-search-relevance/issues/144#issuecomment-1718315467

Reply:
Trying to find a way to make it similar to the mockups, but I'm not sure if it's possible with `EuiPageHeader`. I was taking a look at the [documentation](https://oui.opensearch.org/1.3/#/layout/page-header), there doesn't seem to be a `prop` to specify position of the description.